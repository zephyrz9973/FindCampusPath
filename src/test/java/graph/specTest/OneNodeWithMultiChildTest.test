#This is to test the behavior when one parent have multiple child

#Create a graph
CreateGraph graph1

#Add some nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Add edges from n1 to others
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n1 n4 e3

#List nodes of the graph
ListNodes graph1

# Print the nodes in the graph and the outgoing edges from n1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3
ListChildren graph1 n4