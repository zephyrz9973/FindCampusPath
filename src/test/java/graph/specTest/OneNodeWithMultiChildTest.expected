#This is to test the behavior when one parent have multiple child

#Create a graph
created graph graph1

#Add some nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Add edges from n1 to others
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n1 to n4 in graph1

#List nodes of the graph
graph1 contains: n1 n2 n3 n4

# Print the nodes in the graph and the outgoing edges from n1
the children of n1 in graph1 are: n2(e1) n3(e2) n4(e3)
the children of n2 in graph1 are:
the children of n3 in graph1 are:
the children of n4 in graph1 are: